pages/index.js

```diff
@@ -16,30 +18,15 @@ export default function Home(props) {
       </Head>
 
       <main className={styles.main}>
-        <Image
```

Comment:
Inlined this image as it is not important to the tutorial/demo and was taking up visual real estate.